### ISSUE TITLE
Cannot redock FilesAsFolders window

### ISSUE THREAD

**Issue by yaminb** [dail8859/NotepadNext#451]
Steps to reproduce:
1.  Check View-FolderAsWorkspace
2. Undock the FolderAsWorkspace Window 
3. You cannot redock it on the left pane as it original

**Comment by dail8859** [comment]
What OS and Notepad Next version are you using?

On Windows 10 and Notepad Next v0.6.3 it works fine for me.

**Comment by dail8859** [comment]
Thanks for the info! I don't have access to any Linux platforms currently...hopefully someone else will be able to test this out.

**Comment by yaminb** [comment]
I downloaded qtcreator. Ill play around with it to see if i can figure it
out. Might be a flag or somesthing.

**Comment by dail8859** [comment]
@yaminb Any help would be greatly appreciated! 😄

**Comment by yaminb** [comment]
Might be a qt issue actually.

https://github.com/frescobaldi/frescobaldi/issues/1273

Potentially fixed i. Qt 6.7.

**Comment by dail8859** [comment]
That's unfortunate to hear it is a Qt bug, but at least there is a known reason now! Thanks for looking into that!

**Comment by yaminb** [comment]
Might be a silly question, but I'm using QT Creator. It came with QT 6.3.1.
Do you know how/where to download 6.7 or higher? I can try it and test it out to make sure that is fixed.
Surprisingly hard to find.

**Comment by dail8859** [comment]
I simplified it down a bit to just the `findChildren()` call and the loop and tested it on Windows.

If the QDockWidget is drug out it will "snap" back to the original position, or it will be dropped on whatever valid area it was hovering over. My assumption is that Qt _temporarily_ creates a floating window for the animation. Possibly another Qt bug related to the floating widgets? Maybe turn off `QDockWidget::DockWidgetMovable` so it cannot be clicked and dragged.

Here is what your suggested code changes look like on Windows for reference:
![dockwidget](https://github.com/dail8859/NotepadNext/assets/3694843/f9a42d02-59d5-4ff0-98c6-57108a2b4f68)

**Comment by yaminb** [comment]
Yeah, QT is just not playing nice with wayland. Probably just wait till 6.7 release of QT and see if that fixes all this.

**Comment by dail8859** [comment]
@yaminb Thanks so much for looking into this and attempting at a possible workaround! We'll wait for v6.7 and keep our fingers crossed 🤞

**Comment by agoss94** [comment]
I also did have this issue or a related one on fedora40 Workstation. I was able to revert the view by deleting the frescobaldi config file in ~/.config/frescobaldi/frescobaldi.conf. After this I had at least the default view back.

**Comment by Lampe2020** [comment]
> Probably just wait till 6.7 release of QT and see if that fixes all this.

Well, I'm on Qt 6.10.2 and experience the same issue in Okteta (I have never heard about NotepadNext before researching this bug, but if it's a bug in Qt itself the exact application shouldn't matter) on KWin_wayland of Plasma 6.6.2 on Fedora 43. So it does not look like it is fixed yet…
